src/redux/reducers/signUp.ts

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+interface signUpState {
+  email: string,
+  password: string
+}
+
+const initialState: signUpState = {
+  email: '',
+  password: ''
```

Comment:
why we need store password in store? 1st its not secure, 2-d it does not make sence